test/common/quic/envoy_quic_h3_fuzz_helper.h

```diff
@@ -0,0 +1,98 @@
+#include <set>
+
+#include "test/common/quic/envoy_quic_h3_fuzz.pb.h"
+
+#include "quiche/quic/core/crypto/null_decrypter.h"
+#include "quiche/quic/core/crypto/null_encrypter.h"
+#include "quiche/quic/core/quic_connection.h"
+#include "quiche/quic/core/quic_versions.h"
+
+namespace Envoy {
+namespace Quic {
+
+// This class serializes structured protobuf `HTTP/3` messages to bytes as they
+// would be sent over the wire.
+class H3Serializer {
+public:
+  H3Serializer(std::set<uint32_t>& streams) : open_h3_streams_(streams){};
+  size_t serialize(char* buffer, size_t buffer_len, bool unidirectional, uint32_t type, uint32_t id,
```

Comment:
Please comment about this public interface.

Instead of passing in c-type char array, can you make it return std::string?